src/player.cpp

```diff
@@ -5438,6 +5433,13 @@ void player::suffer()
             }
         }
 
+        if( ( has_trait( trait_NARCOLEPTIC ) || has_artifact_with( AEP_SCHIZO ) ) ) {
+            if( one_in( to_turns<int>( 8_hours ) ) ) {
+                add_msg( m_bad, _( "It's a good time to lie down and sleep." ) );
```

Comment:
The message seems a bit misleading (almost implying the player should sleep more), maybe something more like:

"You're suddenly overcome with the urge to sleep and you pass out."